internal/tools/aws/database.go

```diff
@@ -23,7 +23,9 @@ import (
 )
 
 const mysqlConnStringTemplate = "mysql://%s:%s@tcp(%s:3306)/mattermost?charset=utf8mb4,utf8&readTimeout=30s&writeTimeout=30s"
+const mysqlConnReaderStringTemplate = "%s:%s@tcp(%s:3306)/mattermost?readTimeout=30s&writeTimeout=30s"
```

Comment:
Note to reviewers that these constants should probably be moved to helper methods like for multitenant databases, but didn't want to refactor that in this PR. Will clean up in a future PR.